roxctl/common/client.go

```diff
@@ -72,6 +72,7 @@ func GetRoxctlHTTPClient(am auth.Method, timeout time.Duration, forceHTTP1 bool,
 	retryClient.HTTPClient.Transport = transport
 	retryClient.HTTPClient.Timeout = timeout
 	retryClient.RetryWaitMin = 10 * time.Second
+	retryClient.Logger = nil
```

Comment:
Why?

Reply:
Noticed when using the HTTP client it now printed out a `DEBUG` log when accessing some URL in the command output; so removed it.
We also did this with cloud sources to not pollute logs (and in this case especially to not have this printed to stdout).